tests/dashbio_demos/app_ideogram.py

```diff
@@ -14,833 +15,750 @@
+            id='height-input',
+            min=1,
+            max=10,
+            value=3,
+            className='ideogram-slider'
+        ),
+        html.Div(className='ideogram-controls-name', children='Orientation'),
+        dcc.Dropdown(
+            className='ideogram-dropdown',
+            id='orientation-anote',
+            options=[
+                {
+                    'label': 'Vertical',
+                    'value': 'vertical',
+                },
+                {
+                    'label': 'Horizontal',
+                    'value': 'horizontal',
+                },
+            ],
+            value='horizontal',
+        ),
+    ]
+
+}
+
+ideograms_initial = {
+    'custom': dict(
+        id='ideo-custom',
+        dataDir='https://unpkg.com/ideogram@1.3.0/'
+        'dist/data/bands/native/',
+        orientation='horizontal',
+        organism='human',
+        chrHeight=300,
+        chrMargin=10,
+        chrWidth=8,
+        rotatable=True,
+    ),
+
+    'homology': dict(
+        id='ideo-homology',
+        showBandLabels=True,
+        showChromosomeLabels=True,
+        showFullyBanded=True,
+        fullChromosomeLabels=True,
+        chrHeight=400,
+        chrMargin=200,
+        rotatable=False,
+        perspective='comparative',
+    ),
+    'brush': dict(
+        id='brush-ideo',
+        dataDir='https://unpkg.com/ideogram@1.3.0/'
+        'dist/data/bands/native/',
+        organism='human',
+        chromosomes=['1'],
+        brush='chr1:1-10000000',
+        chrHeight=900,
+        resolution=550,
+        orientation='horizontal',
+    ),
+    'annotations': dict(
+        id='ideo-annotations',
+        dataDir='https://unpkg.com/ideogram@1.3.0/'
+        'dist/data/bands/native/',
+        organism='human',
+        assembly='GRCh37',
+        orientation='horizontal',
+        showBandLabels=True,
+        chrHeight=275,
+        chrMargin=28,
+        rotatable=True,
+        filterable=True,
+        className='ideogram-custom',
     )
-    def orientation_change(orientation):
-        return orientation
+}
 
-    # Chr Width
 
-    @app.callback(
-        Output("ideo-custom", "chrWidth"),
-        [Input("chr-width-input", "value")]
-    )
-    def chr_width(value):
-        return value
+def layout():
+    return html.Div(id='ideogram-body', children=[
+        html.Div(id='ideogram-container'),
+        html.Div(id='ideogram-control-tabs', children=[
+            dcc.Tabs(id='ideogram-tabs', children=[
+                dcc.Tab(
+                    label='About',
+                    value='what-is',
+                    children=html.Div(className='ideogram-tab', children=[
+                        html.H4('What is Ideogram?'),
+                        html.P('Ideogram is a tool used to schematically '
+                               'represent chromosomes. Bands on the chromosomes '
+                               'can show the locations of specific genes.'),
```

Comment:
```suggestion
                               'show the location of specific genes.'),
```

Reply:
What's the issue with "locations", exactly?

Reply:
Grammar (each gene has one location, not multiple locations) or statistics (location seen as a variable, which has many possible values)...

Reply:
I thought it would be fine, since "genes" is pluralized too!

Reply:
It's not terribly wrong but the issue (weakness) with this wording is that it includes the semantic possibility that each gene could have multiple locations. Since this is never the case (as far as I know and I'm more than happy to learn), why not go for stricter wording? Again, the tool is meant for *scientific* contexts.